dash_table/package-info.json

```diff
@@ -37,50 +37,49 @@
   },
   "author": "Chris P <chris@plot.ly",
   "license": "MIT",
-  "dependencies": {
-    "@types/react": "^16.4.8",
-    "ramda": "^0.25.0",
-    "react": "16.3.0",
-    "react-dom": "16.3.0",
-    "react-select": "^1.2.1",
-    "sheetclip": "^0.3.0"
-  },
+  "dependencies": {},
   "devDependencies": {
-    "@babel/cli": "^7.1.2",
-    "@babel/core": "^7.1.2",
+    "@babel/cli": "^7.1.5",
+    "@babel/core": "^7.1.6",
     "@babel/polyfill": "^7.0.0",
-    "@babel/preset-env": "^7.1.0",
+    "@babel/preset-env": "^7.1.6",
     "@babel/preset-react": "^7.0.0",
-    "@cypress/webpack-preprocessor": "^2.0.1",
-    "@percy-io/percy-storybook": "^2.0.0",
-    "@storybook/cli": "^4.0.7",
-    "@storybook/react": "^4.0.7",
-    "@types/ramda": "^0.25.36",
+    "@cypress/webpack-preprocessor": "^4.0.2",
+    "@percy-io/percy-storybook": "^2.1.0",
+    "@storybook/cli": "^4.0.8",
+    "@storybook/react": "^4.0.8",
+    "@types/ramda": "^0.25.42",
+    "@types/react": "^16.7.7",
     "@types/react-select": "^1.2.1",
-    "babel-eslint": "^8.2.3",
+    "babel-eslint": "^10.0.1",
     "babel-loader": "^8.0.4",
     "core-js": "^2.5.7",
-    "css-loader": "^0.28.11",
-    "cypress": "^3.1.0",
-    "eslint": "^4.19.1",
-    "eslint-config-prettier": "^2.9.0",
-    "eslint-plugin-import": "^2.12.0",
-    "eslint-plugin-react": "^7.9.1",
-    "file-loader": "^1.1.11",
+    "css-loader": "^1.0.1",
+    "cypress": "^3.1.2",
+    "eslint": "^5.9.0",
+    "eslint-config-prettier": "^3.3.0",
+    "eslint-plugin-import": "^2.14.0",
+    "eslint-plugin-react": "^7.11.1",
+    "file-loader": "^2.0.0",
     "http-server": "^0.11.1",
-    "less": "^3.8.0",
+    "less": "^3.8.1",
     "less-loader": "^4.1.0",
-    "npm": "^6.1.0",
+    "npm": "^6.4.1",
     "npm-run-all": "^4.1.3",
-    "react-docgen": "^2.20.1",
-    "style-loader": "^0.21.0",
-    "ts-loader": "^4.4.2",
+    "ramda": "^0.26.0",
+    "react": "16.6.3",
+    "react-docgen": "^2.21.0",
+    "react-dom": "16.6.3",
+    "react-select": "^1.2.1",
```

Comment:
There is a new version 2.1.2 of this but we can't update because of internal dependencies on this version's structure. See #250